sandbox/apparmor/apparmor.go

```diff
@@ -327,6 +327,9 @@ func probeParserFeatures() ([]string, error) {
 	if tryAppArmorParserFeature(parser, "network qipcrtr dgram,") {
 		features = append(features, "qipcrtr-socket")
 	}
+	if tryAppArmorParserFeature(parser, "capability bpf,") {
+		features = append(features, "bpf")
```

Comment:
Can this featured be named "cap_bpf" (or at least "cap-bpf" so that it is clear this is around the bpf capability, not other bpf related functionality?

Reply:
yes, cap-bpf looks right